lib/ukintctlr/ukintctlr.c

```diff
@@ -203,6 +210,40 @@ void uk_intctlr_irq_handle(struct __regs *regs, unsigned int irq)
 	return;
 }
 
+int uk_intctlr_irq_alloc(unsigned int *irqs, __sz count)
+{
+	unsigned long start, idx;
+
+	start = uk_bitmap_find_next_zero_area(irqs_allocated,
+					      ALLOCABLE_IRQ_COUNT, 0,
+					      count, 0);
+	if (start == ALLOCABLE_IRQ_COUNT)
+		return -ENOSPC;
+
+	uk_bitmap_set(irqs_allocated, start, count);
+	for (idx = start; idx < (start + count); idx++) {
+		*irqs = idx + FIRST_ALLOCABLE_IRQ;
+		irqs++;
+	}
```

Comment:
Newline here for consistency? As `return 0;` is not part of the same logic block of the loop above, right?